ansible_collections/amarao/ip/tests/integration/targets/ip_link_device_attribute/tasks/main.yaml

```diff
@@ -302,3 +302,80 @@
       always:
         - name: TEST4, cleanup interface
           command: ip link delete group 42
+
+    # TEST5: Support for group operations
+    # stages:
+    # - create bridge
+    # - create dummy interface
+    # - run the module in check_mode
+    # - validate output
+    # - run the module for real
+    # - validate output
+    # - run module second time
+    # - check of idempotency
+    # - run the module with nomaster option
+    # - validate output
+    # - delete interfaces
+    - name: TEST5 - work with master/nomaster
+      become: true
+      block:
+        - name: TEST5, preparation, create linux bridge
+          command: ip link add dev bridge-example type bridge
+        - name: TEST5, create dummy interface for test
+          command: ip link add dev dummy-master type dummy
+        - name: TEST5, run module with check_mode
+          check_mode: true
+          ip_link_device_attribute:
+              name: dummy-master
+              master: bridge-example
+              state: up
+          register: res
+        - name: TEST5, assert results for check_mode
+          assert:
+            that:
```

Comment:
I'm not sure it's a good idea to use output from check mode. Theoretically module may return 'expected' change in check mode. It's better to ask module for data in the next task (call ir with name only).